ratis-server/src/test/java/org/apache/ratis/server/impl/MiniRaftCluster.java

```diff
@@ -347,7 +347,6 @@ public void start() throws IOException {
 
     initServers();
     startServers(servers.values());
-
```

Comment:
There still a whitespace change.

Reply:
@133tosakarin , please take a look the file diff, there are two whitespace changes now.

Reply:
Um, I'm sorry, I haven't had time to modify it yet